src/utils/convertKeysToKebabCase.ts

```
@@ -0,0 +1,10 @@
+export const convertKeysToKebabCase = (
+  obj: Record<string, any>,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)